changelog/11868.bugfix.rst

```diff
@@ -0,0 +1 @@
+Reintroduced the default value for the ``ignore_args`` of internal ``FixtureManager.getfixtureclosure``: even though it is an internal API, it was changed by accident in 8.0.0 and this broken a number of plugins.
```

Comment:
It wasn't an accident really, so changing the wording a bit:

```suggestion
Reintroduced the default value for the ``ignore_args`` of internal ``FixtureManager.getfixtureclosure``: it was changed in 8.0.0 and this has broken a number of plugins.
```

Reply:
I meant "non-intentional", meaning that we probably did not mean to drop the default argument and break external usages, figuring it was used internally only.

But I will apply your suggestion anyway, no need to over explain things.